bonfire/processor.py

```diff
@@ -224,7 +224,7 @@ def _sub_image_tags(self, items):
         content = json.dumps(items)
         for image, image_tag in self.image_tag_overrides.items():
             # easier to just re.sub on a whole string
-            content, subs = re.subn(rf"{image}:\w+", rf"{image}:{image_tag}", content)
+            content, subs = re.subn(rf"{image}:[^\"]*", rf"{image}:{image_tag}", content)
```

Comment:
As we discuessed lets use `rf"{image}:[-\w\.]+"` to match any word character OR a dash OR a period -- which should be the only valid characters we would encounter in an image tag string.

Reply:
Agree, done.